Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_16_11.md

```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### MDE Malware - Incident Enrichment
+
+- Added a task to verify that the incident fields necessary to set alerts table info to the layout are not empty.
```

Comment:
In case we will change that to validate the evidence array

`Added a task to ensure that Evidence information should be displayed in the "Alerts and Related info".`

Reply:
done